backend/data/seed.js

```diff
@@ -26,9 +29,16 @@ async function seedConfigs() {
 // Seed function
 export async function seedDatabase() {
   try {
-    // only connect if not testing otherwise connect to test db
-    let mongoURI = process.env.MONGO_URI + "/cdj"; // pulled from compose.yaml env vars
-    if (process.env.NODE_ENV === 'test') { mongoURI = mongoURI + '-test'; }
+    // Set the URI for the database
+    let mongoURI = process.env.MONGO_URI + "/cdj";
+
+    // If testing, use the in-memory database
+    if (memoryMongo) { 
+      console.log("Setting up in-memory database...")
```

Comment:
Nitpick: this line is missing a semicolon. Sorry for missing it 🤣

Reply:
Alright, I'm sure there's a lot more of these kinds of problems. I'm going to update the eslintrc.cjs file with some additional rules, set up the linter in my IDE then fix them all in one go. I was considering doing it in a separate PR but I think the sooner the better so I stop making these kinds or mistakes 😆

Reply:
@davidzlu Backend should be nice and pretty now 🙂